aws-credentials/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
All the vars used in this role should be defined with default values in a role-specific default vars file ([example](https://github.com/Chia-Network/ansible-roles/blob/main/chia-blockchain/defaults/main.yml))